rust/hyperlane-base/src/types/multisig.rs

```diff
@@ -38,39 +42,48 @@ impl MultisigCheckpointSyncer {
         // Get the latest_index from each validator's checkpoint syncer.
         let mut latest_indices = Vec::with_capacity(validators.len());
         for validator in validators.iter() {
-            let addr = H160::from(*validator);
-            if let Some(checkpoint_syncer) = self.checkpoint_syncers.get(&addr) {
+            let address = H160::from(*validator);
+            if let Some(checkpoint_syncer) = self.checkpoint_syncers.get(&address) {
                 // Gracefully handle errors getting the latest_index
-                if let Ok(Some(index)) = checkpoint_syncer.latest_index().await {
-                    latest_indices.push(index);
+                match checkpoint_syncer.latest_index().await {
+                    Ok(Some(index)) => {
+                        trace!(?address, ?index, "Validator returned latest index");
+                        latest_indices.push(index);
+                    }
+                    err => {
+                        debug!(?address, ?err, "Failed to get last index from validator");
```

Comment:
nit: `last` -> `latest` would be nice just to keep language consistent

Reply:
Done